sockeye/utils.py

```diff
@@ -241,45 +241,41 @@ def variance(self) -> float:
         else:
             return self._M2 / self._count
 
-
-def smallest_k(matrix: np.ndarray, k: int,
-               only_first_row: bool = False) -> Tuple[Tuple[np.ndarray, np.ndarray], np.ndarray]:
+def topk(scores: mx.nd.NDArray, k: int, batch_size: int, t: int, use_mxnet_topk: bool):
     """
     Find the smallest elements in a numpy matrix.
 
-    :param matrix: Any matrix.
-    :param k: The number of smallest elements to return.
+    :param scores: Vocabulary scores for the next beam step. (beam_size * batch_size, target_vocabulary_size)
```

Comment:
lets define the shape here as `(batch_size * beam_size, target_vocabulary_size)`